### ISSUE TITLE
Please Provide an option to disable animation

### ISSUE THREAD

**Issue by theprantadutta** [quelchx/vue-writer#4]
First of all, thank you for creating this awesome library.

**Describe the bug**
Sometimes, we want to stop the animation after it ran once. Then we just want to show the cursor, but I can't seem to do that with this library. Also, I do want to run the animation only when the element is scrolled to, not before, and not after.

**To Reproduce**

I can't find an option to do this behavior. Is there any other way I can do this?

**Expected behavior**

I want to run the typewriter animation only once when I scrolled to the element.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: windows
 - Browser chrome, safari
 - Version 1.1.2

**Smartphone (please complete the following information):**
 - Device: Galaxy S10 Lite
 - OS: android
 - Browser chrome
 - Version 1.1.2

**Additional context**

**Comment by quelchx** [comment]
This package was originally intended to loop for my personal use case. 

Saying that I can look into adding this feature in time.

I closed this issue -- but reopened it as a enhancement I intend to add at a later time.

**Comment by quelchx** [comment]
This enhancement may take time to add. Currently I'm finishing my last semester of school and have been very busy. 

I should be able to add a prop that you can pass to toggle the animation repeating or not. 

I looked into this today off my one computer but cloning the packages and installing  them seemed to caused issues of my Windows PC. 

Later this week I will attempt to add this feature with the original code that currently sits on my other computer.

Thanks for the feedback!

**Comment by theprantadutta** [comment]
Thanks for the fast reply. I will be waiting to hear from you.

**Comment by quelchx** [comment]
> Thanks for the fast reply. I will be waiting to hear from you.

I added a small feature to run the animation through however many times the user wishes. By default, it will loop forever unless you specify. I added documentation to show how to use this feature. 

An example of this would be as follows:
```html
<vue-writer :array="['Hello', 'World'] :iterations='1'">
```

If you don't add the iterations param, it's by default 0 and will loop forever.

I hope this is what you were seeking as an addition, sorry it took so long for the update -- my week at school has been hectic. 
Cheers!
